docs/interpreter/flink.md

```diff
@@ -148,13 +148,18 @@ You can also set other flink properties which are not listed in the table. For a
   <tr>
     <td>zeppelin.flink.planner</td>
     <td>blink</td>
-    <td>planner or flink table api, blink or flink</td>
+    <td>planner of flink table api, blink or flink</td>
```

Comment:
I maybe would put `flink` and `blink` into some emphasis tags, maybe `<tt>` because these are specific values.

Reply:
`<tt>` is deprecated, so I use `<em>` instead.